pkg/github/prcreation/prcreation.go

```diff
@@ -84,7 +84,7 @@ func (o *PRCreationOptions) UpsertPR(localSourceDir, org, repo, branch, prTitle,
 			return fmt.Errorf("failed to fork %s/%s: %w", org, repo, err)
 		}
 		repo = forkName
-		if err := waitForRepo(username, repo, o.GithubClient); err != nil {
+		if err := WaitForRepo(username, repo, o.GithubClient); err != nil {
```

Comment:
why this is now exposed?

Reply:
I am trying to resue that function (here)[https://github.com/openshift/ci-tools/pull/1237/files#diff-4440cf6f38e6026b7a8028aa83c9f664R115]